css/print.css

```diff
@@ -84,119 +418,200 @@ emu-figure img {
+
+/* 8.3.3 avoid breaking in small alg lists */
+#sec-static-semantics-containsundefinedcontinuetarget li,
+
+/* 9.1.1.3 attempt to fit two tables on the same page */
+#table-additional-fields-of-function-environment-records,
+
+/* 9.1.1.4 attempt to fit two long tables on the same page */
+#table-additional-fields-of-global-environment-records,
+
+/* 12.2 5-row table should be able to fit on one page */
+#table-white-space-code-points,
+
+/* 12.3 4-row table should be able to be on one page */
+#table-line-terminator-code-points,
+
+/* 14.7.5.10.3 short table shouldn't be broken */
+#table-for-in-iterator-instance-slots,
+
+/* 15.7.1 keep header with list to avoid awkward page break */
+#sec-class-definitions-static-semantics-early-errors > emu-grammar:first-of-type + ul,
+
+/* 15.7.8 keep third grammar together with alg */
+#sec-static-semantics-privateboundidentifiers > emu-grammar:nth-of-type(3)
+{
+  break-inside: avoid-page;
 }
-@page intro {
-  --page-number-style: lower-roman;
+
+/* 12.10.1 long-ass note can break wherever it wants */
+#sec-rules-of-automatic-semicolon-insertion > emu-note {
+  break-before: initial;
+  break-inside: initial;
 }
 
-#toc {
-  counter-reset: page 1;
+/* 12.10.2, 12.10.3, 12.10.3.2 Sections start with an <em> not inside a <p> */
+#sec-examples-of-automatic-semicolon-insertion > em,
+#sec-interesting-cases-of-automatic-semicolon-insertion > em,
+#sec-asi-cases-with-no-lineterminator-here > em {
+  display: block;
+  margin-top: 1.25ex;
 }
-#spec-container > emu-clause:first-of-type {
-  counter-reset: page 1;
+
+/* 13.5.1.1 the vast, vast majority of the document, we are better off with emu-grammar being block. this is the exception. */
+#sec-delete-operator-static-semantics-early-errors emu-grammar {
+  display: inline-block;
 }
 
-@page :left {
-  @bottom-left {
-    content: counter(page, var(--page-number-style));
-  }
+/* 15.3 A very long term combined with inline-block, nowrap, and justified text resulting in weird punctuation */
+#_ref_20642 {
+  font-size: 98%;
 }
-@page :right {
-  @bottom-right {
-    content: counter(page, var(--page-number-style));
-  }
+
+/* 15.7.1 keep header with list to avoid awkward page break */
+#sec-class-definitions-static-semantics-early-errors > emu-grammar:first-of-type {
+  break-after: avoid-page;
 }
 
-@page :first {
-  @bottom-left {
-    content: '';
-  }
-  @bottom-right {
-    content: '';
-  }
+/* 16.2.1.5.4 squish margins on graphics so table 46 fits on one page */
+#sec-example-cyclic-module-record-graphs figure:has(> img) {
+  margin: 0;
+}
+#sec-example-cyclic-module-record-graphs figure > img {
+  margin: 0;
+}
+#sec-example-cyclic-module-record-graphs figure:has(> img) > figcaption {
+  margin-bottom: 1ex;
 }
+#sec-example-cyclic-module-record-graphs emu-figure:has(> figure > img) + p {
+  margin-top: 1ex;
+}
+
+/* 16.2.1.5.4 table 46 is still wide after pivoting */
+#table-module-graph-cycle-async-fields-1 table {
+  font-size: 95%;
+}
```

Comment:
Could you just `colspan` the `[[Status]]` row all the way across so that `~evaluating-async~` doesn't stretch out every column?